src/main/java/leto/parser/Parser.java

```diff
@@ -0,0 +1,35 @@
+package leto.parser;
+
+import java.util.Scanner;
+
+import static leto.ui.Ui.shortSay;
+
+public class Parser {
+    private static Scanner sc = new Scanner(System.in);
+
+    public Parser() {
+    }
+
+    public static void readCommandAndExecute() {
+        boolean bye = false;
```

Comment:
Could be `isBye` or `isExit`

Reply:
Will fix upstream! Thanks for the lookout